src/code/z_actor.c

```diff
@@ -1386,21 +1388,21 @@ void func_8002ED80(Actor* actor, GlobalContext* globalCtx, s32 flag) {
 }
 
 PosRot* func_8002EEE4(PosRot* arg0, Actor* actor) {
```

Comment:
Same deal with these 2 functions, probably something like `Actor_GetFocus` (or whichever name we go with)